### ISSUE TITLE
Do not optimal ActiveQuery::exists checking

### ISSUE THREAD

**Issue by TheHett** [yiisoft/yii2-mongodb#177]
https://github.com/yiisoft/yii2-mongodb/blob/master/ActiveQuery.php#L161
```php
namespace yii\mongodb;
class ActiveQuery extends Query implements ActiveQueryInterface
{

    /**
     * @inheritdoc
     */
    public function exists($db = null)
    {
        return $this->one($db) !== null;
    }
}
```

this approach:
- load largest data from mongodb than needed
- populate record and do not use it
- (possibly) Loading  relatated models (if no lazy relations specified)

**Comment by klimov-paul** [comment]
Resolved by commit eb06029dc54515bfb5d5bf61f55fe3693c623d8e

**Comment by TheHett** [comment]
Thanks!
